### PR TITLE
KOGITO-9291 Build multiplatform

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-9291

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -155,31 +155,16 @@ podman-build: test ## Build container image with the manager.
 docker-push: ## Push docker image with the manager.
 	docker push ${IMG}
 
-# PLATFORMS defines the target platforms for  the manager image be build to provide support to multiple
```
Comment (desmax74) [inline Makefile:158]: This was added following the operator SDK instructions, can we leave for the future upgrades of the sdk ?

File: .ci/jenkins/Jenkinsfile.promote [hunk 2]
```diff
@@ -18,8 +18,8 @@ pipeline {
         KOGITO_CI_EMAIL_TO = credentials("${JENKINS_EMAIL_CREDS_ID}")
 
         // Linked to node label
-        CONTAINER_ENGINE='podman'
-        CONTAINER_ENGINE_TLS_OPTIONS='--tls-verify=false'
+        CONTAINER_ENGINE = 'podman'
```
Comment (desmax74) [inline .ci/jenkins/Jenkinsfile.promote:21]: Why we have podman in the promote and docker in the deploy ?

Comment (radtriste) [reply .ci/jenkins/Jenkinsfile.promote:21]: in deploy, we need `docker` for the multiplatform build
in promote, we need the a container engine only for login to registry, so `podman` should be the default

File: Makefile (hunk 1 above)
Comment (radtriste) [reply Makefile:158]: will set back.
Just did not want it to be confusing

**Review by ricardozanini — APPROVED** [review]

**Review by spolti — APPROVED** [review]

**Pull request merged**